app/services/dependent_flow_service.rb

```diff
@@ -31,6 +31,21 @@ def show?
     when "Ctc::Questions::Dependents::ClaimChildAnywayController"
       eligibility = child_qualification(except: :claimable_test)
       dependent.cant_be_claimed_by_other_no? && eligibility.qualifies?
+    when "Ctc::Questions::Dependents::RelativeMemberOfHouseholdController"
+      return false if child_qualification.qualifies?
+
+      eligibility = relative_qualification(except: [:residence_test, :financial_support_test, :claimable_test])
+      eligibility.qualifies? && eligibility.requires_member_of_household_test?
+    when "Ctc::Questions::Dependents::RelativeFinancialSupportController"
+      return false if child_qualification.qualifies?
```

Comment:
To see qualifying relative pages, you'll need to have failed the qualifying child test.

Reply:
This is what ensures that the "other descendant of sibling" will see this page if they fail the age qualification.